src/components/PartnerCardMobile/index.css

```diff
@@ -0,0 +1,26 @@
+@import "../../styles/colors.css";
+@import "../../styles/layout.css";
+
+.root {
+  display: flex;
+  padding: $spacing-16;
+
+  background-color: white;
+
+  box-shadow: 8px 8px 28px rgba(0, 0, 0, 0.08);
```

Comment:
Use the new color function. color(black alpha(8%))

Reply:
👍